orbit/pkg/migration/readwriter.go

```diff
@@ -57,7 +57,7 @@ func (rw *ReadWriter) RemoveFile() error {
 }
 
 func (rw *ReadWriter) GetMigrationType() (string, error) {
-	data, err := rw.read()
+	data, err := rw.read() // TODO: confirm error handling with jahziel, what about other errors?
```

Comment:
Good point! We should return the other errors from `read`. I'll add that to my PR.